src/libtmux/session.py

```diff
@@ -253,6 +253,9 @@ def new_window(
         if window_name:
             window_args += ("-n%s" % window_name,)
 
+        if window_name == '':
+            window_args += ('-n', '')
```

Comment:
@trankchung Are you open to adding a test?

Reply:
@tony Yes I’ll add a test for this tomorrow.

Reply:
@trankchung Sounds good!